app/lib/screens/history/history.dart

```diff
@@ -11,6 +18,36 @@ class _TravelHistoryState extends State<TravelHistory> {
     print('Init: Travel history');
   }
 
+  final pdf = pw.Document(); //Create a blank document
+  //add data into the pdf file
+  buildPdf() {
```

Comment:
I happen to see this in the build output now:
```
flutter: Helvetica has no Unicode support see https://github.com/DavBfr/dart_pdf/wiki/Fonts-Management
```

- [ ] Please fix or find out why

Reply:
@holtzmak @mansiPaTeLL I believe this is because the font Helvetica is not included in pubspec.yaml

Reply:
@holtzmak @MummenRider I tried to fixing this by using suggestion guide. But then it is giving me some exception handling error. I think this can be considered as a separate task as it takes some more research.

Reply:
@mansiPaTeLL , try this [solution](https://github.com/DavBfr/dart_pdf/issues/196)

Reply:
@MummenRider Thanks for this solution. I tried and zI having an issues.